README.md

```
@@ -44,15 +44,23 @@ func main() {
 	if err != nil {
 		panic(err)
 	}
-	defer w.Close()
+	defer func() {
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)